From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Otto=20Pfl=C3=BCger?= <otto.pflueger@abscue.de>
Date: Sat, 17 May 2025 20:51:36 +0200
Subject: [PATCH 101/225] dt-bindings: timer: add compatibles for Unisoc
 UMS9230
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The UMS9230 timers are compatible with the SC9860 timers.

Signed-off-by: Otto Pflüger <otto.pflueger@abscue.de>
---
 .../bindings/timer/sprd,sc9860-timer.yaml         | 15 ++++++++++++---
 1 file changed, 12 insertions(+), 3 deletions(-)

diff --git a/Documentation/devicetree/bindings/timer/sprd,sc9860-timer.yaml b/Documentation/devicetree/bindings/timer/sprd,sc9860-timer.yaml
index 62c6da8ba..528d55df1 100644
--- a/Documentation/devicetree/bindings/timer/sprd,sc9860-timer.yaml
+++ b/Documentation/devicetree/bindings/timer/sprd,sc9860-timer.yaml
@@ -19,9 +19,18 @@ description:
 
 properties:
   compatible:
-    enum:
-      - sprd,sc9860-timer
-      - sprd,sc9860-suspend-timer
+    oneOf:
+      - items:
+          - enum:
+              - sprd,ums9230-suspend-timer
+          - const: sprd,sc9860-suspend-timer
+      - items:
+          - enum:
+              - sprd,ums9230-timer
+          - const: sprd,sc9860-timer
+      - enum:
+          - sprd,sc9860-timer
+          - sprd,sc9860-suspend-timer
 
   reg:
     maxItems: 1
